### ISSUE TITLE
[prosiebenmaxx] unable to extract clip id

### ISSUE THREAD

**Issue by SonGokuSSJ** [rg3/youtube-dl#15290]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.01.14*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [X] I've **verified** and **I assure** that I'm running youtube-dl **2018.01.14**

### Before submitting an *issue* make sure you have:
- [X] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [X] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [X] Bug report (encountered problems with youtube-dl)
- [X] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add the `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):

```

C:\Users\BrollyLSSJ\Downloads\InuYasha>youtube-dl.exe -a p7maxx.txt --verbose
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-a', 'p7maxx.txt', '--verbose']
[debug] Batch file urls: ['https://www.prosiebenmaxx.de/anime/naruto-shippuden/v
ideo/9297-folge-297-wuensche-des-vaters-und-liebe-der-mutter-ganze-folge', 'http
s://www.prosiebenmaxx.de/anime/naruto-shippuden/video/112-folge-12-das-maedchen-
das-ein-junge-ist-ganze-folge', 'https://www.prosiebenmaxx.de/anime/detektiv-con
an/videos/52-folge-114-das-grab-im-garten-2-ganze-folge', 'https://www.prosieben
maxx.de/anime/detektiv-conan/videos/51-folge-113-das-grab-im-garten-1-ganze-folg
e', 'https://www.prosiebenmaxx.de/anime/one-piece/video/9530-folge-530-der-koeni
g-der-fischmenscheninsel-neptune-gott-des-meeres-ganze-folge']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2018.01.14
[debug] Python version 3.4.4 (CPython) - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg 3.3.3, ffprobe 3.3.3
[debug] Proxy map: {}
[prosiebensat1] anime/naruto-shippuden/video/9297-folge-297-wuensche-des-vaters-
und-liebe-der-mutter-ganze-folge: Downloading webpage
ERROR: Unable to extract clip id; please report this issue on https://yt-dl.org/
bug . Make sure you are using the latest version; type  youtube-dl -U  to update
. Be sure to call youtube-dl with the --verbose flag and include its complete ou
tput.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpo17hktrm\bu
ild\youtube_dl\YoutubeDL.py", line 784, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpo17hktrm\bu
ild\youtube_dl\extractor\common.py", line 438, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpo17hktrm\bu
ild\youtube_dl\extractor\prosiebensat1.py", line 442, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpo17hktrm\bu
ild\youtube_dl\extractor\prosiebensat1.py", line 387, in _extract_clip
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpo17hktrm\bu
ild\youtube_dl\extractor\common.py", line 803, in _html_search_regex
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpo17hktrm\bu
ild\youtube_dl\extractor\common.py", line 794, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract clip id; please report th
is issue on https://yt-dl.org/bug . Make sure you are using the latest version;
type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose fl
ag and include its complete output.


C:\Users\BrollyLSSJ\Downloads\InuYasha>
...
<end of log>
```

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
https://www.prosiebenmaxx.de/anime/naruto-shippuden/video/9297-folge-297-wuensche-des-vaters-und-liebe-der-mutter-ganze-folge
https://www.prosiebenmaxx.de/anime/naruto-shippuden/video/112-folge-12-das-maedchen-das-ein-junge-ist-ganze-folge
https://www.prosiebenmaxx.de/anime/detektiv-conan/videos/52-folge-114-das-grab-im-garten-2-ganze-folge
https://www.prosiebenmaxx.de/anime/detektiv-conan/videos/51-folge-113-das-grab-im-garten-1-ganze-folge
https://www.prosiebenmaxx.de/anime/one-piece/video/9530-folge-530-der-koenig-der-fischmenscheninsel-neptune-gott-des-meeres-ganze-folge

Note that **youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free)**. In order for site support request to be accepted all provided example URLs should not violate any copyrights.

---

### Description of your *issue*, suggested solution and other information

prosiebenmax has updated their website and now downloads arent working anymore. Yesterday it was working fine (I was using 2017.12.2 until this bug occurred today). Then I switched to 2018.01.14, but the problem is still present.

**Comment by phihag** [comment]
I believe this issue to be fixed with 2a3683c3780698dfa1d1175734fca97d1cd6c215. If the problem persists with youtube-dl 2018.01.19 or newer, please post here so that we can reopen this issue.

**Comment by SonGokuSSJ** [comment]
Yes, it works again. Thank you very much.
